### ISSUE TITLE
/action messages sent via Slackbot as well as in channels

### ISSUE THREAD

**Issue by me-and** [aeirola/slack-irc-client#22]
When someone uses the `/me does a thing` syntax in Slack and IRC, it's presented as that user performing an action. With slack-irc-client, though, the actions come entirely separate from the channel they happen in as messages from Slackbot.

For example, in Slack, I have a direct message from Slackbot that looks like this, with no reference to the channel the action happened in:

> **CTCP notice from me_and**
> ACTION waves

Normal IRC clients, however, show in the correct channel, for example:

```
me_and | here are some words in a normal message
     * | me_and waves
```

And regular Slacks format `/me` commands with italics, also in the correct title:

> **me_and**
> here are some words in a normal message
> **me_and**
> *waves*

slack-irc-client should convert `ACTION` CTCP messages into regular messages that appear in the correct channel from the correct person, and formatted in italic to match actions in regular Slack.

I've not checked how slack-irc-client handles outgoing `/me`-style messages, as I don't currently have an easy set-up to do so, but if it isn't the case already, it should convert the thing to a CTCP `ACTION` message for sending back to the IRC server.

**Comment by me-and** [comment]
Correction! The messages do appear in the channels as I'd expect, they just **also** appear as DMs from Slackbot.

**Comment by aeirola** [comment]
I suppose this is the same issue as mentioned in #13, and is fixed by #16
